### ISSUE TITLE
[Feature Request] Input Overlay

### ISSUE THREAD

**Issue by jjallen37** [frankborden/slippilab#47]
The controller input displays should be visible as an overlay in the replay video area instead of being a tab. I want to view inputs while selecting different clips from the "Clips" tab,

This tool is incredible, great work!

**Comment by frankborden** [comment]
Yeah finding a good layout has been tricky. I used to have it on the replay area but it was really too crowded for doubles. I might try shrinking the replay down more like YouTube pages. In the meantime there is a keyboard shortcut for cycling through clips at least, click the keyboard icon for a list.
